docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -55,68 +55,125 @@ If you are installing Rancher on a K3s cluster with **Raspbian Buster**, follow
 
-## CPU and Memory
+In particular, requirements on this page are subject to "typical use" conditions, which include:
+ - total count of Kubernetes resources under 60 thousand per type
+ - up to 120 pods per node
+ - up to 200 CRDs in the upstream (local) cluster
+ - up to 100 CRDs in downstream clusters
+ - up to 50 Fleet deployments
+Higher numbers are possible but requirements might be higher.
+Note that visualization in the UI might be impacted for users with visibility on all resources in a type above 20 thousand total items.
 
-Hardware requirements scale based on the size of your Rancher deployment. Provision each individual node according to the requirements. The requirements are different depending on if you are installing Rancher in a single container with Docker, or if you are installing Rancher on a Kubernetes cluster.
+:::note Evolution:
 
-### RKE and Hosted Kubernetes
+Rancher's code base evolves, use cases change, and the body of accumulated Rancher experience grows every day.
 
-These CPU and memory requirements apply to each host in the Kubernetes cluster where the Rancher server is installed.
+Because of that, this document's recommendations are subject to change over time with the aim of providing increasingly more concrete and accurate guidelines.
 
-These requirements apply to RKE Kubernetes clusters, as well as to hosted Kubernetes clusters such as EKS.
+If you find your Rancher deployment does not comply with recommendations in this document, while it used to comply with earlier revisions, you can [contact Rancher](https://rancher.com/contact/) for an ad-hoc evaluation.
 
-| Deployment Size | Clusters   | Nodes        | vCPUs  | RAM     |
-| --------------- | ---------- | ------------ | -------| ------- |
-| Small           | Up to 150  | Up to 1500   | 2      | 8 GB    |
-| Medium          | Up to 300  | Up to 3000   | 4      | 16 GB   |
-| Large           | Up to 500  | Up to 5000   | 8      | 32 GB   |
-| X-Large         | Up to 1000 | Up to 10,000 | 16     | 64 GB   |
-| XX-Large        | Up to 2000 | Up to 20,000 | 32     | 128 GB  |
+:::
+
+
+### RKE2 Kubernetes
 
-Every use case and environment is different. Please [contact Rancher](https://rancher.com/contact/) to review yours.
+Minimum CPU and memory requirements for each individual node in the [Kubernetes cluster Rancher is installed in](install-upgrade-on-a-kubernetes-cluster.md) are listed in the table below.
+
+Please note that a highly available setup with at least 3 nodes is required for all production usages.
+
+| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
```

Comment:
```suggestion
| Managed Infrastructure Size | Clusters  | Nodes      | vCPUs | RAM   |
```

Instead of "Up to" in each cell, revise to have the header row state "Maximum # of Clusters" & "Maximum # of Nodes"

Reply:
Addressed as of 548e4de67ae, resolving.